lib/proofer/vendor/mock.rb

```diff
@@ -130,6 +135,10 @@ def fail_confirmation_with_bad_phone(session_id)
         )
       end
 
+      def fail_resolution_with_exception(_uuid)
+        raise 'Failed to contact proofing vendor'
```

Comment:
I wondered that too. Ambivalent. Didn't really want to introduce a specific exception class just for this, since vendor-specific drivers could conceivably throw any kind of exception.